test/unit/rules/no-invalid-role-test.js

```diff
@@ -74,5 +74,30 @@ generateRuleTests({
         column: 0,
       },
     },
+
+    {
```

Comment:
Can you also add one valid case showing that an invalid role is allowed when `catchInvalidRoles` is off?